ssz_serialization/merkleization.nim

```diff
@@ -464,7 +464,9 @@ template writeBytesLE(chunk: var array[bytesPerChunk, byte], atParam: int,
   chunk[at ..< at + sizeof(val)] = toBytesLE(val)
 
 func chunkedHashTreeRoot[T](
-    merkleizer: var SszMerkleizerImpl, arr: openArray[T]): Digest =
+    totalChunks: static Limit, arr: openArray[T]): Digest =
```

Comment:
The code was written in the way it was written to avoid unnecessary code bloat.
Previously, `chunkedHashTreeRoot` will be instantiated exactly once per type. Now, you are instantiating it once per unique input size.

Reply:
You are right. I'll have to find a solution for this that will work well with the multi-merkle code, without bloating the code. Will revise this (currently, 4 different copies for `NU8*` are created).

Reply:
I'll postpone this commit and bundle a less code-bloaty variant with the merkle multiproof functionality.